### ISSUE TITLE
Optimizer: ADAM vs Momentum SGD

### ISSUE THREAD

**Issue by insikk** [insikk/ml_study_note#1]
Many literature in computer vision deep learning still favors momentum SGD. I saw most of my colleges choose ADAM as defacto standard recently. 
Why such litterateurs still stick to the Momentum SGD?

Obvious downside of Momentum SGD is we need to set hyperparameter and manually schedule the learning rate. 

https://arxiv.org/abs/1705.08292 provide evidence that Momentum SGD generalize well.
